s3/replication/common/src/s3replicationcommon/job.py

```diff
@@ -234,6 +236,12 @@ def get_operation_type(self):
         return self._obj["source"]["operation"]["type"]
 
     # Source attribute accessors
+    def get_object_tagset(self):
+        """
+        Get object tagset
+        """
+        return self._obj["User-Defined-Tags"]
```

Comment:
is it inside fdmi record?

Reply:
yes, it will come as a part of fdmi record if we add tag put request then.